src/azul/project/hca/transformers.py

```diff
@@ -23,26 +23,56 @@ def _project_dict(bundle: api.Bundle) -> dict:
     }
 
 
-def _specimen_dict(biomaterials: Mapping[api.UUID4, api.Biomaterial]) -> List[Mapping[str, Any]]:
+def _specimens_dict(biomaterials: Mapping[api.UUID4, api.Biomaterial]) -> List[Mapping[str, Any]]:
```

Comment:
This method will go away, but for the record, the value type is too general, it should be `SpecimenFromOrganism`, not `Biomaterial`.